src/main/java/no/sikt/nva/pubchannels/handler/fetch/publisher/FetchPublisherByIdentifierAndYearHandler.java

```diff
@@ -0,0 +1,43 @@
+package no.sikt.nva.pubchannels.handler.fetch.publisher;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.sikt.nva.pubchannels.handler.PublicationChannelClient;
+import no.sikt.nva.pubchannels.handler.fetch.FetchByIdentifierAndYearHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+import static nva.commons.core.attempt.Try.attempt;
+
+public class FetchPublisherByIdentifierAndYearHandler extends FetchByIdentifierAndYearHandler<Void,
```

Comment:
I think it's easier to read if the extends starts on a new line.

So that you get the whole extends visible.

`extends FetchByIdentifierAndYearHandler<Void, FetchByIdAndYearResponse> `

Reply:
Done in new commit